src/proto/package/package.proto

```diff
@@ -35,8 +35,7 @@ message GetPackageResponse {
 service PackageService {
   rpc GetPackage (GetPackageRequest) returns (GetPackageResponse) {
```

Comment:
This package needs to return the package tags and the User information.

<img width="900" alt="image" src="https://github.com/goravel-ecosystem/market-backend/assets/24771476/a6878400-d43f-40aa-bed4-3321a6e3f29e">

Reply:
This PR is good, we can leave it here for now, and implement https://goravel.atlassian.net/browse/GM-25 first, then we can get tags in this PR.

Reply:
Sorry ~ I forgot the relationship between these two tickets when you picked up this ticket.

Reply:
How is GM-25 is related to this issue? There we are searching a tag by their name but here we want to get all related tags to a package.

Reply:
The GetTags endpoint can add a filter field: package_id, it can return all tags of a package. Optimized the description. 

<img width="503" alt="image" src="https://github.com/goravel-ecosystem/market-backend/assets/24771476/b57b6e2f-7385-435b-b617-be11ccd2863b">